raiden-ts/src/channels/epics.ts

```diff
@@ -607,14 +613,24 @@ const makeDeposit$ = (
   approveNonce?: number,
 ): Observable<channelDeposit.failure> => {
   if (!deposit?.gt(Zero)) return EMPTY;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function stopRetryIfRejected(err: any, count: number) {
```

Comment:
This is always `throw`n from above, but not necessarily an `Error`: ethers/Metamask usually don't create Error instances, but instead return an object like `{ count: number; message: string; ...details }`, so it's not an `Error` and therefore I can't type it like that here. But it may be actual `Error`s.

Reply:
Ah okay. That is said. But since we have no influence on that...
Thanks for explaining it. :pray: